File: src/main/java/fi/metatavu/metaform/server/xlsx/XlsxException.java
package fi.metatavu.metaform.server.xlsx;
/**
 * Exception throw when XLSX generation fails
 * 
 * @author Antti Leppä
 */
public class XlsxException extends Exception {

  private static final long serialVersionUID = 8611634313163534478L;

  /**
   * Constructor
   * 
   * @param message message
   * @param original original exception
   */
  public XlsxException(String message, Throwable original) {
    super(message, original);
  }
  
  /**
   * Constructor
   * 
   * @param message message
   */
  public XlsxException(String message) {
    super(message);
  }
  
}

File: src/main/kotlin/fi/metatavu/metaform/server/exceptions/XlsxException.kt
package fi.metatavu.metaform.server.exceptions

/**
 * Exception throw when XLSX generation fails
 *
 * @author Antti Leppä
 */
class XlsxException : Exception {
    /**
     * Constructor
     *
     * @param message message
     * @param original original exception
     */
    constructor(message: String?, original: Throwable?) : super(message, original)

    /**
     * Constructor
     *
     * @param message message
     */
    constructor(message: String?) : super(message)

    companion object {
        private const val serialVersionUID = 8611634313163534478L
    }
}